website/docs/opengl/c1/3-hello-texture.md

```diff
@@ -347,7 +347,7 @@ And now, some extra content to go with this tutorial!
 If you get curious about how the texture parameters work, let's learn it now!
 
 First, let's learn the structure of the command:
-```c#
+```c
```

Comment:
```suggestion
```cs
```

Reply:
Lol, I should've used "replace all"

Reply:
Lol imagine use replace all abd change everything to c